src/cppsim/state_dm.hpp

```diff
@@ -440,7 +440,7 @@ class DensityMatrixCpu : public QuantumStateBase {
 };
 
 typedef DensityMatrixCpu
-    DensityMatrix; /**< QuantumState is an alias of QuantumStateCPU */
+    DensityMatrix; /**< QuantumState is an alias of QuantumStateCpu */
```

Comment:
これ、元のコメントの意図がこうなのかな、と思いました

```suggestion
    DensityMatrix; /**< DensityMatrix is an alias of DensityMatrixCpu */
```